src/main/resources/application.yml

```diff
@@ -89,6 +89,16 @@ nakadi:
     checkRunMs: 600000 # 10 min
     timelineCleanup.runPeriodMs: 3600000 # 1 hour
     consumerNodesCleanup.runPeriodMs: 21600000 # 6 hours
+  hashPartitioning.overrideOrder:
```

Comment:
I removed it from here, will place it to private configuration
https://github.com/zalando/nakadi/pull/687/commits/e9193762008a31488fac413d651580ea7ebd6a3a